app/controllers/resources_controller.rb

```diff
@@ -1,17 +1,36 @@
 class ResourcesController < ApplicationController
   def index
     category_id = params[:category_id]
-    resources = if category_id
-                  Resource.joins(:categories).where('categories.id' => category_id)
-                else
-                  Resource.all
-                end
+    result = if category_id
+               resources.joins(:categories).where('categories.id' => category_id)
+             else
+               Resource.all
+             end
 
-    render json: resources.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    render json: result.to_json(resource_inclusion)
   end
 
   def show
-    resource = Resource.find(params[:id])
-    render json: resource.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    resource = resources.find(params[:id])
+    render json: resource.to_json(resource_inclusion)
+  end
+
+  private
+
+  def resources
+    Resource.includes(:addresses, :phones, :categories,
+                      schedule: :schedule_days)
+  end
+
+  def resource_inclusion
+    schedule_attrs = { include: :schedule_days }
+    service_attrs = [:notes, schedule: schedule_attrs]
```

Comment:
Oh, yeah. Good catch! Bad copy-paste there; this should have been wrapped in a hash as you're doing. Looks like we need a test for the new nested models, since the specs we've got didn't catch it.